packages/react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecManager.java

```diff
@@ -651,6 +653,21 @@ public void run() {
         } else if (commandType.equals("blur")) {
             parent.clearFocusFromJS();
             return;
+        } else if (commandType.equals("onMarkFormatting")) {
+            String colorString;
+            Boolean resetColor;
+
+            if (args != null && args.getString(0) != null) {
+                colorString = args.getString(0);
+            } else {
+                colorString = "";
+            }
+
+            parent.onMarkFormatting(colorString);
+            return;
+        } else if (commandType.equals("onRemoveMarkFormatting")) {
+            // This is handled by setActiveFormats
+            return;
```

Comment:
This appears to be a no-op. If so, why does it exist?

Reply:
Since it's only available on iOS, I thought we could add some information on why this isn't available on Android.